vtr_flow/tasks/regression_tests/vtr_reg_basic/basic_no_timing/config/config.txt

```diff
@@ -5,15 +5,23 @@
 # Path to directory of circuits to use
 circuits_dir=benchmarks/verilog
 
+# Path to directory of includes circuits to use
+includes_dir=benchmarks/verilog/include
+
 # Path to directory of architectures to use
 archs_dir=arch/no_timing/memory_sweep
 
 # Add circuits to list to sweep
+circuit_list_add=ch_intrinsics_modified.v
```

Comment:
How does this config work?  Are the two include files listed below pasted into each benchmark file before it is run? 
Do the two generic_definitions.vh files have meaning only for ch_intrinsics_modified?
If they have meaning only for ch_intrinsics_modified, this would be better tested by making a new regression test called hdl_include (or something like that) that only checks that ch_intrinsics works.  If that test fails, it is now much more obvious where to start looking for a bug.

So if my understanding is correct I think you should
1. split this into two tests
2. comment the config.txt file more extensively.